docs/security-council-mgmt.md

```diff
@@ -176,15 +176,17 @@ It accepts proposals in the same format that normal governors do, however it ove
 
 Voting and proposing can occur using the standard governance UIs.
 
-### 9/12 Security Council
+### Non-Emergency Security Council
```

Comment:
This is only consistent with constitution because we're also making the non emergency governor a 9/12 multisig - so in the future we would need to be careful if we ever reduced the threshold again to move these roles back to the emergency governor.

I dont know how we can guard against. Could do:
1. Add a big comment somewhere about the threshold - but I dont know where we could put a comment that it would get picked up in this case
2. Add a check somewhere in the sec council manager that wont allow these updates to be called unless its by a safe with 9/12 threshold.
3. Add a guard into the non emergency safe that doesnt allow the threshold to be reduced unless a special string is provided eg keccak256("the member roles on the sec council manager have already been revoked")

Reply:
we could put something in `gotchas.md` or `security-council-manager.md` but i'm not sure they would be noticed

Reply:
i actually think documenting it should be sufficient, the nonemergency can only do 2 things: queue l2 timelock and call the manager, so when the threshold is changed I would think this would be remembered

Reply:
Ok, lets stick it in the gotchas or something then